vm/src/obj/objtuple.rs

```diff
@@ -292,27 +312,11 @@ pub fn init(context: &PyContext) {
 tuple(iterable) -> tuple initialized from iterable's items
 
 If the argument is a tuple, the return value is the same object.";
+    PyTuple::extend_class(context, tuple_type);
+
     extend_class!(context, tuple_type, {
-        "__add__" => context.new_rustfunc(PyTupleRef::add),
-        "__bool__" => context.new_rustfunc(PyTupleRef::bool),
-        "__eq__" => context.new_rustfunc(PyTupleRef::eq),
-        "__ne__" => context.new_rustfunc(PyTupleRef::ne),
-        "__contains__" => context.new_rustfunc(PyTupleRef::contains),
-        "__getitem__" => context.new_rustfunc(PyTupleRef::getitem),
-        "__hash__" => context.new_rustfunc(PyTupleRef::hash),
-        "__iter__" => context.new_rustfunc(PyTupleRef::iter),
-        "__len__" => context.new_rustfunc(PyTupleRef::len),
         (slot new) => tuple_new,
-        "__mul__" => context.new_rustfunc(PyTupleRef::mul),
-        "__rmul__" => context.new_rustfunc(PyTupleRef::rmul),
-        "__repr__" => context.new_rustfunc(PyTupleRef::repr),
-        "count" => context.new_rustfunc(PyTupleRef::count),
-        "__lt__" => context.new_rustfunc(PyTupleRef::lt),
-        "__le__" => context.new_rustfunc(PyTupleRef::le),
-        "__gt__" => context.new_rustfunc(PyTupleRef::gt),
-        "__ge__" => context.new_rustfunc(PyTupleRef::ge),
         "__doc__" => context.new_str(tuple_doc.to_string()),
```

Comment:
`__doc__` can be replaced by doc comment of struct. See https://github.com/RustPython/RustPython/pull/895/files#diff-bf99aa0e3eaf69e3d751384530066205R18-R26